docs/UserGuide.md

```diff
@@ -7,151 +7,255 @@
+![help message](images/helpMessage.png)
 
-* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
+### Save Your Data 💾
+* All changes are saved automatically
+* No manual saving needed
+* Data is stored in your LegacyLink folder
 
-* All commands are **case-sensitive**. <br>
-e.g. The command `Help` is different from `help` and therefore, results in an `Unknown command` error
-</panel>
+### Exit LegacyLink 👋
+**Format:**
+```
+exit
+```
+* Closes the application
+* All data is automatically saved
+> ⚠️ **Warning:** No confirmation message is displayed. 
 
-### Viewing help : `help`
+### Editing The Data File
 
-Shows a message explaining how to access the help page.
+LegacyLink data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json and [JAR file location]/data/eventbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
-Format: `help`
+<box type="warning" seamless>
+
+**Caution:**
+If your changes to the data file makes its format invalid, LegacyLink will discard all data and start with an empty data file at the next run.  Hence, it is recommended to take a backup of the file before editing it.<br>
+Furthermore, certain edits can cause the LegacyLink to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
+
+</box>
 
-![help message](images/helpMessage.png)
 
-### Listing all persons : `list -p`
 
-Shows a list of all persons in the address book.
+## Managing Contacts
 
-Format: `list -p`
+### View Your Contact List 👥
+```
+list -p
+```
+* Shows all your family contacts in one view
+* Use this to check contact details or before using other commands
+
+![ListPersonCommand.png](images/ListPersonCommand.png)
 
 <box type="tip" seamless>
 
 **Tip:** Preceding, trailing and intermediate whitespaces will still result in a successful command.
 </box>
 
-![ListPersonCommand.png](images/ListPersonCommand.png)
+### Add a New Contact ➕
+**Format:**
+```
+add -n NAME -p PHONE_NUMBER -e EMAIL -rs RELATIONSHIP
+```
+* Adds a person to the address book.
+* All parameters `NAME`, `PHONE_NUMBER`, `EMAIL`, `RELATIONSHIP` must be present.
 
-### Adding a person: `add`
+<box type="tip" seamless>
 
-Adds a person to the address book.
+**Rules to Remember:**
 
-Format: `add -n NAME -p PHONE_NUMBER -e EMAIL -rs RELATIONSHIP`
+| Field       | Requirements                                                                 |
+|-------------|---------------------------------------------------------------------------|
+| Name        | • Letters, numbers, and spaces only<br>• Cannot be blank                  |
+| Phone       | • 3-15 digits<br>• Numbers only                                           |
```

Comment:
Change to "Phone: 3-15 digits, but alphabetical characters (A-Z, a-z) and special characters (/ - _ ( )) are permitted. At least one digit is required in the number"